recipes/gtabview/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gtabview" %}
+{% set version = "0.8" %}
+{% set hash_type = "sha256" %}
+{% set hash = "959160e6ee65110fb43af8fdc7e83cbb7469b91f90ab25b63c611241c7bf5fb1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - gtabview = bin.cli.main:main
```

Comment:
Looking at the `setup.py` it does not look like the the cli uses an entry point; it's just a script in the `bin` directory. So this section is not needed. I'm not sure though if anything else has to be added to the recipe to move the cli to the correct location. I'll dig around and see if I can find any relevant documentation.